syncstorage-db/src/tests/db.rs

```diff
@@ -16,9 +19,9 @@ lazy_static! {
 }
 
 #[tokio::test]
-async fn bso_successfully_updates_single_values() -> Result<()> {
+async fn bso_successfully_updates_single_values() -> Result<(), DbError> {
```

Comment:
Bit surprised you didn't create a type for this, just to save keystrokes, but it's fine.

Reply:
There were a few different `Result`/`DbResult`/`DbFuture` aliases, and I thought it would make things easier to follow if we were more explicit with our return types but 🤷🏻 perhaps that's just me